### ISSUE TITLE
VS Code no longer filling screen

### ISSUE THREAD

**Issue by techiemac** [blinksh/blink#1781]
### Checklist

- [X] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [X] Read the [docs](http://blink.sh/docs/)
- [X] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

iPad Pro version 16.4.1(a)
Blink version v16.1.7.748 (App Store)
VS Server version 20220909.27

### Describe the bug

When I launch code on my iPad pro, it’s no longer filling the screen. I’m using stage manager but it’s quite pronounced when I go to fullscreen mode.

![IMG_1764](https://github.com/blinksh/blink/assets/632991/68c05493-d042-4bf2-ad08-520acbd7608d)

**Comment by techiemac** [comment]
Odd, this was happening for the past several hours but now seems to have resolved itself.

**Comment by carloscabanero** [comment]
Hi! Do you mind to send us a full-screen screenshot of the whole window? I cannot really see properly what you meant there. It looks like maybe misbehavior with the bottom bar?

![IMG_0034](https://github.com/blinksh/blink/assets/536729/7187836a-8e22-4fca-94c9-30fe14568aa4)

**Comment by techiemac** [comment]
That’s actually a full screenshot. You are correct, it’s something going on with the lower bar. FYI, I was in a limited bandwidth situation when I took the screenshot (airplane wifi). I seem to recall that VSCode is run locally in a full frame webview so the low bandwidth situation may be a red herring. 
I saw this behavior when running `code-server` on my build instance and just running code with no `code-server` (screenshot is from the latter).
